### DISCUSSION TITLE
每日单词

### DISCUSSION THREAD

**Discussion by user19970225** [YiiGuxing/TranslationPlugin#5557] [Q&A]
每日单词能不能手动打开，现在只能每次重启idea才会打开

**Comment by YiiGuxing** [comment]
要手动打开每日单词有两种办法：

1. 为打开每日单词操作添加快捷键，然后通过快捷键打开；
  ![image](https://github.com/YiiGuxing/TranslationPlugin/assets/10737066/f2d1e283-6dcf-4bd0-bb7e-59748c90ecf5)

2. 通过单词本中的按钮打开。
  ![image](https://github.com/YiiGuxing/TranslationPlugin/assets/10737066/4f514bf0-11b8-40d2-b641-e6cce78a8687)
